src/citrine/resources/design_execution.py

```diff
@@ -22,15 +22,15 @@ class DesignExecution(Resource['DesignExecution'], Pageable, AsynchronousObject)
     Possible statuses are INPROGRESS, SUCCEEDED, and FAILED.
     Design executions also have a ``status_description`` field with more information.
 
-    Parameters
-    ----------
-    project_id: str
-        Unique identifier of the project that contains the workflow execution
-
     """
 
     _paginator: Paginator = Paginator()
     _collection_key = 'response'
+    # This should really be _session, but _fetch_page assumes there is a 'pageable' parameter
+    session: Optional[Session] = None
```

Comment:
Why can't we rename this one?

Reply:
> There's one place where its `session` still (no `_`). What would it take to rename that one?

I renamed it. The `_fetch_page` method is more flexible than I realized initially